cfme/web_ui/__init__.py

```diff
@@ -1957,7 +1957,8 @@ def click_path(self, *path, **kwargs):
         See :py:meth:`expand_path` for more informations about synopsis.
         """
         node = self.expand_path(*path, **kwargs)
-        sel.click(node)
+        if node is not None:
```

Comment:
This inherently changes the behavior, have you audited usages of click_path() to handle the case where the node returned is None? Before these callers wouldn't have had to account for that fact, since this would throw an exception if you tried to call sel.click(None).